.eslintrc-js.js

```diff
@@ -1,4 +1,4 @@
-{
```

Comment:
What is the reasoning behind this change?

Reply:
I tend to prefer them because they allow comments, imports an environment variables. i wanted it to keep the `.eslintrc-ts.js` as `.js` so that the comments could be left in, and i figured i'd move this to `.js` for consistency. Happy to change it if you prefer `json` though

Reply:
Nah, your point is fair enough, happy to keep it as js